src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,73 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React, { memo } from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  selectStatus: string,
+  onSelectStatus: (event: React.ChangeEvent<HTMLSelectElement>) => void,
+  query: string,
+  onQueryChange: (value: string) => void,
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: React.FC<Props> = memo(({
+  selectStatus,
+  onSelectStatus,
+  query,
+  onQueryChange,
+}) => {
+  const handleSubmit = (event: React.ChangeEvent<HTMLFormElement>) => {
```

Comment:
I agree with @olena-yanovska that it is no need to create methods here, beacause we do not have additional logic in our functions

it is not critical - you can leave it here, but consider it for the future

Reply:
got it, thanks)